src/components/Filters/CheckboxOption.tsx

```diff
@@ -137,7 +138,7 @@ export function CheckboxOption(props: CheckboxOptionProps): JSX.Element | null {
           onChange={handleChange}
           disabled={isOptionsDisabled}
         />
-        <label className={labelClasses} htmlFor={optionId}>{labelText}</label>
+        <label className={labelClasses} htmlFor={optionId}>{String(labelText)}</label>
```

Comment:
do we need this change?

Reply:
I see. since the error is for NumberRangeValue, what does it look like in the UI if we coerce labelText of type NumberRangeValue to be a string? if it looks ok, maybe we can call toString when defining labelText. if it doesn't, we should either omit the display name or come up with an alternative that looks better

Reply:
i did `toString` and it seems to work fine in the test site